README.md

```diff
@@ -1,7 +1,18 @@
-# avr-libstdcpp: libstdc++ port for avr-gcc
-[![Build Status](https://github.com/modm-io/avr-libstdcpp/actions/workflows/compile_examples.yml/badge.svg)](https://github.com/modm-io/avr-libstdcpp/actions)
+avr-libstdcpp: libstdc++ port for avr-gcc
 ==================
 
+<p align="center">
+    <a href="https://github.com/modm-io/avr-libstdcpp/actions">
+        <img src="https://github.com/modm-io/avr-libstdcpp/actions/workflows/compile_examples.yml/badge.svg" alt="Build Status"></a>
+    <a href="https://github.com/modm-io/avr-libstdcpp/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc">
+        <img src="https://custom-icon-badges.herokuapp.com/github/issues-raw/modm-io/avr-libstdcpp?logo=github" alt="Issues" /></a>
+    <a href="https://github.com/modm-io/avr-libstdcpp/blob/master/COPYING3">
+        <img src="https://img.shields.io/badge/license-GPL%203.0-blue.svg" alt="GNU GENERAL PUBLIC LICENSE 3.0"></a>
+    <a href="https://github.com/modm-io/avr-libstdcpp">
+        <img src="https://img.shields.io/github/languages/code-size/modm-io/avr-libstdcpp" alt="GitHub code size in bytes" /></a>
+</p>
+
+
```

Comment:
I don't see the benefit of this change and consider it bad practise to include icons/badges from third party domains.

Reply:
I'm completely agnostic on it. The original repo had the build-success badge only. We can happily go back to that if that is the preference.